tests/unit/lambda_debug_mode/test_config_parsing.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+
+from localstack.utils.lambda_debug_mode.lambda_debug_mode_config import (
+    load_lambda_debug_mode_config,
+)
+
+DEBUG_CONFIG_EMPTY = ""
+
+DEBUG_CONFIG_NULL_FUNCTIONS = """
+functions:
+    null
+"""
+
+DEBUG_CONFIG_NULL_FUNCTION_CONFIG = """
+functions:
+  arn:aws:lambda:eu-central-1:000000000000:function:functionname:$LATEST:
+    null
+"""
+
+DEBUG_CONFIG_NULL_DEBUG_PORT = """
+functions:
+  arn:aws:lambda:eu-central-1:000000000000:function:functionname:
+    debug-port: null
+"""
+
+DEBUG_CONFIG_NULL_TIMEOUT_DISABLE = """
+functions:
+  arn:aws:lambda:eu-central-1:000000000000:function:functionname:
+    debug-port: null
+    enforce-timeout: null
```

Comment:
The flag is now called `enforce-timeouts`, not `enforce-timeout`. So we are not testing the `null` value here, but actually the default, as pydantic per default ignores extra fields.